### PR TITLE
Update optee pages

### PR DESCRIPTION
**Pull request by jbech-linaro**
This is a general update and should hopefully also take care of the bad links mentioned in #46 . Due to this I believe it is better to merge this PR first, then rebase #46 and then merge that.

// Joakim

### CODE REVIEW

File: docs/ti.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: OP-TEE on Texas Instruments SoCs
 permalink: "/docs/ti/"
-layout: container-breadcrumb
-description: The instructions here will tell how to run OPTEE on Texas Instruments
+layout: default
```
Comment (kylekirkby) [inline docs/ti.md]: Why is this changed to default ? The default layout isn't as expected :) It's just the wire frame for other layouts. Sorry for the confusion.

File: _data/nav.yml [hunk 2]
```diff
@@ -20,8 +20,6 @@ pages:
             url: /about/
           - text: Security Working Group
             url: /swg/
-          - text: Committee
```
Comment (kylekirkby) [inline _data/nav.yml]: Did you intend to remove this from the Navigation Bar ?

File: docs/ti.md (hunk 1 above)
Comment (jbech-linaro) [reply docs/ti.md]: Oh, that was unintentionally, I had a merged conflict in that region when rebasing to master. I'll change it back, thanks for spotting this one.

File: _data/nav.yml (hunk 2 above)
Comment (jbech-linaro) [reply _data/nav.yml]: Yes I did, we have no Committee in OP-TEE so it makes no sense to have it there.
